packages/react/src/modules/guide/components/Banner/Banner.tsx

```diff
@@ -3,11 +3,7 @@ import clsx from "clsx";
 import React from "react";
 
 import { maybeNavigateToUrlWithDelay } from "../helpers";
-import {
-  ButtonContent,
-  TargetButton,
-  TargetButtonWithGuideContext,
-} from "../types";
+import { ButtonContent, TargetButton, TargetButtonWithGuide } from "../types";
```

Comment:
Unrelated, but I changed the type name to `TargetButtonWithGuide` from `TargetButtonWithGuideContext`